libcontainer/specconv/example.go

```diff
@@ -41,11 +41,6 @@ func Example() *specs.Spec {
 					"CAP_KILL",
 					"CAP_NET_BIND_SERVICE",
 				},
-				Ambient: []string{
```

Comment:
The same here.

Reply:
This one is not an example, but the data used by `runc spec` (and thus in all runc integration tests).

In the other comment you say it's better to turn on inheritable capabilities, but the whole point of https://github.com/opencontainers/runc/security/advisories/GHSA-f3fp-gc8g-vw66 was to NOT set inheritable capabilities by default. By extension, ambient capabilities should not be set by default either (as they won't work without inheritable ones).